docs/deploying-airbyte/on-gcp-compute-engine.md

```diff
@@ -91,59 +61,53 @@ sudo apt-get install -y docker-ce docker-ce-cli containerd.io
 sudo usermod -a -G docker $USER
 ```
 
-* Install `docker-compose`
+7. Install docker-compose on your VM instance terminal by following the below commands in your ssh session on the instance terminal.
 
 ```bash
-# In your ssh session on the instance terminal
 sudo apt-get -y install wget
 sudo wget https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$(uname -s)-$(uname -m) -O /usr/local/bin/docker-compose
 sudo chmod +x /usr/local/bin/docker-compose
 docker-compose --version
 ```
 
-* Close the ssh connection to ensure the group modification is taken into account
+8.  Close the ssh connection on your VM instance terminal to ensure the group modification is taken into account by following the below command in your ssh session on the instance terminal.
 
 ```bash
-# In your ssh session on the instance terminal
 logout
 ```
 
-## Install & start Airbyte
+## Install and launch Airbyte
 
-* Connect to your instance
+To install and launch Airbyte:
+
+1. In your local terminal, connect to your Google Cloud instance
 
 ```bash
-# In your workstation terminal
 gcloud --project=$PROJECT_ID beta compute ssh $INSTANCE_NAME
 ```
 
-* Install Airbyte
+2. In your ssh session on the instance terminal, install Airbyte
 
 ```bash
-# In your ssh session on the instance terminal
 mkdir airbyte && cd airbyte
 wget https://raw.githubusercontent.com/airbytehq/airbyte/master/{.env,docker-compose.yaml}
 docker-compose up -d
 ```
 
 ## Connect to Airbyte
 
-:::danger
-
-For security reasons, we strongly recommend to not expose Airbyte publicly. Future versions will add support for SSL & Authentication.
-
+:::caution
+Warning: For security reasons, we strongly recommended not exposing Airbyte publicly.
 :::
 
-* Create ssh tunnel.
+1. In your local terminal, create an SSH tunnel to connect the GCP instance to Airbyte
 
 ```bash
-# In your workstation terminal
 gcloud --project=$PROJECT_ID beta compute ssh $INSTANCE_NAME -- -L 8000:localhost:8000 -N -f
 ```
 
-* Just visit [http://localhost:8000](http://localhost:8000) in your browser and start moving some data!
+2. Verify the connection by visiting [http://localhost:8000](http://localhost:8000) in your browser
```

Comment:
Add period (.) at the end of the sentence

Reply:
Done